OPHD/States/MapViewStateTurn.cpp

```diff
@@ -202,18 +201,17 @@ void MapViewState::updateMorale()
 	mCurrentMorale -= structuresDisabled;
 	mCurrentMorale -= structuresDestroyed;
 
-
 	mCurrentMorale = std::clamp(mCurrentMorale, 0, 1000);
 
-	// Provide reason values for PopulationPanel
+	mPopulationPanel.clearMoraleReasons();
 	if (birthCount > 0) { mPopulationPanel.addMoraleReason("Births", birthCount); }
 	if (deathCount > 0) { mPopulationPanel.addMoraleReason("Deaths", -deathCount); }
```

Comment:
You could potentially even initialize a new morale reasons list, and then assign it to the population panel at the end. That would have the effect of simultaneously clearing the text list of morale reasons, and any running tally of morale modifiers it might accumulate.